mmdet3d/configs/fcos3d/__init__.py

```diff
@@ -0,0 +1 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
This file should not be kept

Reply:
Thank you for your review, but I found that all other folders contain `__init__.py`. Should I delete this file?